docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>W also needs to be determined by this latent variable Z. Currently it looks like W is defined in terms of C. For latent variable analysis, we'd like to infer something about an unknown parent of both C and X. Currently, we also aren't assessing anything about the CBNs relate to the latent Z, so I wouldn't call this a latent variable analysis. We can either adapt it or remove it. We also should redefine the probabilistic model from the introduction to include Z if we want to add this (see the NOTMAD paper for an example)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>